i18n/en/en.yaml

```
@@ -69,7 +69,7 @@ home_show_less:
 home_falco_what_is_falco_title:
   other: Detect security threats in real time
 home_falco_what_is_falco:
-  other: Falco is the open source solution for runtime security for hosts, containers, Kubernetes and the cloud. Get real-time visibility into abnormal behaviors and potential security threats, intrusions, and data theft or compliance violations.
+  other: Falco, a cloud-native security tool for Linux, employs custom rules on kernel events, enriched with container and Kubernetes metadata, to alert in real time. Enhance your off-host analysis and gain visibility into abnormal behavior, security threats, and compliance violations for comprehensive runtime security.
```

I know we're trying to be concise with this paragraph, but I think we can expand it a little to help with clarity.
```suggestion
  other: Falco is a cloud-native security tool designed for Linux systems. It employs custom rules on kernel events, which are enriched with container and Kubernetes metadata, to provide real-time alerts. By enhancing your off-host analysis, Falco helps you gain visibility into abnormal behavior, potential security threats, and compliance violations, contributing to comprehensive runtime security.
```

---

Thank you, Nate! I like your suggestions, and I think it works for the homepage. On top of that, we are evaluating removing `By enhancing your off-host analysis` from this message and putting a separate reference for that somewhere on the homepage for better clarity. I'm working on it.